airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.tsx

```diff
@@ -168,8 +170,10 @@ const ServiceForm: React.FC<ServiceFormProps> = (props) => {
         );
       }
     });
-    return selectedServiceDefinition?.documentationUrl;
-  }, [availableServices, selectedConnectorDefinitionSpecification]);
+    setDocumentationUrl(selectedServiceDefinition?.documentationUrl ?? "");
```

Comment:
`ServiceForm` is a base component the other connector creation forms are built on top of, so quite a bit of logic was removed by moving all of the opening and URL setting logic into here.